### ISSUE TITLE
Function mcrypt_create_iv() is deprecated

### ISSUE THREAD

**Issue by andrii-trush** [gladchinda/keygen-php#7]
```
file : "/var/www/html/vendor/gladcodes/keygen/src/Keygen/Generators/RandomByteGenerator.php"
line : 51
message : "Function mcrypt_create_iv() is deprecated"
```

My code

`Keygen::bytes(20)->hex()->generate()`

PHP version on production 7.1 - has error

But in my local environment with php 7.2.7 all are ok

**Comment by gladchinda** [comment]
@andrii-trush I have fixed the issue on the `v1.2.0-alpha` branch. I have replaced the deprecated `mcrypt_create_iv()` function with the `random_bytes()` function. It should work correctly without errors now.

**Comment by gladchinda** [comment]
Hello @andrii-trush I have also fixed this issue alongside other bugs in the new `v1.1.2` release of the package. This release also has some improvements and proper unit testing and is fully compatible with the previous versions.

You can install it in your project using:

```sh
$ composer require gladcodes/keygen ~1.1.2
```
